webapp/tests/test_util.py

```diff
@@ -119,3 +121,14 @@ def test_logtime(ok, custom=None, timer=None):
             test_logtime(False)
         self.assertEqual(log.info.call_count, 3)
         self.assertRegexpMatches(log.info.call_args[0][0], r'test :: failed in [-.e0-9]+s')
+
+
+class SafeUnpicklerTest(TestCase):
+
+    def test_load(self):
+        unpickler = util.unpickle()
+        p = b'\x80\x04\x95\r\x00\x00\x00\x00\x00\x00\x00\x8c\ttest.d\xc3\xb8d\x94.'
```

Comment:
I have a possibly better example that may work as intended for both python2.7 and python3: `b"S'test.d\\xc3\\xb8d'\np0\n."`

python2.7:

```
>>> s = 'test.død'
>>> pickle.dumps(s)
"S'test.d\\xc3\\xb8d'\np0\n."
>>> pickle.loads(b"S'test.d\\xc3\\xb8d'\np0\n.")
'test.d\xc3\xb8d'
>>> print(pickle.loads(b"S'test.d\\xc3\\xb8d'\np0\n."))
test.død
```

python3.7:

```
>>> pickle.loads(b"S'test.d\\xc3\\xb8d'\np0\n.")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 6: ordinal not in range(128)
>>> pickle.loads(b"S'test.d\\xc3\\xb8d'\np0\n.", encoding='utf8')
'test.død'
```